Ray's Adventure/raysAdventure/src/mercury/core/levels/BasicLevel.java

```diff
@@ -158,19 +156,37 @@ else if(choice == 3) {
 				//method to list doors in the room
 				ray.getLocation().printDoors();
 				System.out.println();
-				System.out.println("What room would you like to move to?");
-				System.out.println();
-				//Takes numbered input of the door the user would like to go through
-				int roomChoice = Tools.getWholeNumberInput();
-				//method to check door choice is valid
-				while(roomChoice > ray.getLocation().doors.size()){
-					System.out.println("Please select a valid number");
-					roomChoice = Tools.getWholeNumberInput();
+				System.out.println("Would you like to go through a door?");
+				String[] doorChoices = {"Go through a door.", "no."};
+				int doorChoice = Tools.getWholeNumberInput(doorChoices);
+
+<<<<<<< HEAD
```

Comment:
@madeleineclay can you edit your code to remove this line? Also see line 189 in the same file with a similar entry. It is left over from a merge conflict resolved by Github. Make sure the needed code is still present in these sections and not duplicated. Thanks!